frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -167,23 +191,21 @@ export const funnelLogic = kea<funnelLogicType>({
             },
         ],
         propertyCorrelations: [
-            {
-                events: [],
-            } as Record<'events', FunnelCorrelation[]>,
+            { events: [] } as Record<'events', FunnelCorrelation[]>,
             {
                 loadPropertyCorrelations: async () => {
-                    const target_properties =
+                    const targetProperties =
                         values.propertyNames.length === values.allProperties.length ? ['$all'] : values.propertyNames
 
-                    if (target_properties.length === 0) {
+                    if (targetProperties.length === 0) {
                         return { events: [] }
                     }
 
                     const results: Omit<FunnelCorrelation, 'result_type'>[] = (
                         await api.create(`api/projects/${values.currentTeamId}/insights/funnel/correlation`, {
                             ...values.apiParams,
                             funnel_correlation_type: 'properties',
-                            funnel_correlation_names: target_properties,
+                            funnel_correlation_names: values.propertyNames,
```

Comment:
this should be `targetProperties`

Reply:
right. I'm not convinced the special casing is worth it tbh but will update.